apps/web/src/views/Info/components/InfoNav/index.tsx

```diff
@@ -95,13 +95,13 @@ const InfoNav: React.FC<{ isStableSwap: boolean }> = ({ isStableSwap }) => {
   )
 }
 
-const targetChains = [mainnet, bsc, polygonZkEvm, zkSync, arbitrum, linea, base]
+const targetChains = [mainnet, bsc, polygonZkEvm, zkSync, arbitrum, linea, base, opBNB]
 
 export const NetworkSwitcher: React.FC<{ activeIndex: number }> = ({ activeIndex }) => {
   const { t } = useTranslation()
   const chainName = useChainNameByQuery()
   const foundChain = chains.find((d) => d.id === multiChainId[chainName])
-  const symbol = multiChainShortName[foundChain?.id] ?? foundChain?.nativeCurrency?.symbol
+  const symbol = multiChainShortName[foundChain?.id ?? 0] ?? foundChain?.nativeCurrency?.symbol
```

Comment:
use `0` seems different with preview logic

Reply:
How about make it.

`const symbol = foundChain?.id && multiChainShortName[foundChain?.id] ?  multiChainShortName[foundChain.id] : foundChain?.nativeCurrency?.symbol`

Reply:
yeah, that will be right

Reply:
update to -1, this is for fixing ts check